<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1200</width>
      <height>800</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1200</width>
      <height>800</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>600</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>600</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>30</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>战棋游戏</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="btnLevelOne">
              <property name="text">
               <string>LEVEL 1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnLevelTwo">
              <property name="text">
               <string>LEVEL 2</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="MyMapOne" name="widgetMapOne" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>800</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1200</width>
        <height>800</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1200</width>
        <height>800</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="MyMapTwo" name="widgetMapTwo" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>800</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1200</width>
        <height>800</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1200</width>
        <height>800</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSet"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="separator"/>
   <addaction name="actionRule"/>
  </widget>
  <action name="actionSet">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionRule">
   <property name="text">
    <string>规则</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyMapOne</class>
   <extends>QWidget</extends>
   <header>mymapone.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyMapTwo</class>
   <extends>QWidget</extends>
   <header>mymaptwo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
